lib/tests-from-pull-request.js

```diff
@@ -1,32 +1,38 @@
-const fs = require('fs')
-const readline = require('readline')
+module.exports = getTestsFromPR;
 
-module.exports = getTestsFromPR
-
-function getTestsFromPR (context) {
+function getTestsFromPR(context) {
   return Promise.resolve()
-    .then(() => ['Hi Or'])
-  // .then(() => _getTestsFromPR(context))
+    .then(() => _getTestsFromPR(context));
 }
 
-function _getTestsFromPR (context) {
-  const pull = context.issue()
-  const {repos, pullRequests} = context.github
-  // todo: code to get file contect
-  let newTestLines = []
-  let PRfiles
+function _getTestsFromPR(context) {
```

Comment:
I think it's better to merge `getTestsFromPR` and `_getTestsFromPR`

Reply:
style or die :)